.changeset/smart-avocados-smile.md

```diff
@@ -0,0 +1,8 @@
+---
+"@commercetools-frontend/application-components": patch
+"@commercetools-frontend/application-shell": patch
+"@commercetools-frontend/assets": patch
+"@commercetools-local/visual-testing-app": patch
+---
+
+Update illustrations.
```

Comment:
We can also mention maybe which illustrations (by file name) are supposed to be deprecated.

Reply:
Done here - https://github.com/commercetools/merchant-center-application-kit/pull/3017/commits/d9aa1320f66d0b4c590d382ab169a13bcd6bdff3 ✅